include/cantera/kinetics/Falloff.h

```diff
@@ -258,6 +258,65 @@ class SRI : public Falloff
     doublereal m_e;
 };
 
+//! The 1- or 2-parameter Tsang falloff parameterization.
+/*!
+ *  The Tsang falloff model is adapted from that of Troe.
+ *  It provides a constant or linear in temperature value for \f$ F_{cent} \f$:
+ *  \f[ F_{cent} = A + B*T \f]
+ *  
+ *  The value of \f$ F_{cent} \f$ is then applied to Troe's model for the
+ *  determination of the value of \f$ F \f$.
+ *
+ * @ingroup falloffGroup
```

Comment:
Would it be possible to add more context? You provided more detail in the (now eliminated) `ck2cti` changes, i.e.
```python
    """
    The Tsang falloff model is adapted from that of Troe.
    It provides a constant or linear in temperature value for :math:`F_\\mathrm{cent}`
    .. math::
        F_\\mathrm{cent} = A + B*T
    For the Tsang and Troe model the parameter :math:`F` is computed via
    .. math::
        \\log F &= \\left\\{1 + \\left[ \\frac{\\log P_\\mathrm{r} + c}{n - d (\\log P_\\mathrm{r} + c)} \\right]^2 \\right\\}^{-1} \\log F_\\mathrm{cent}
        c &= -0.4 - 0.67 \\log F_\\mathrm{cent}
        n &= 0.75 - 1.27 \\log F_\\mathrm{cent}
        d &= 0.14
    The attributes are:
    =============== ======================= ====================================
    Attribute       Type                    Description
    =============== ======================= ====================================
    `A`             :class:`Quantity`       The :math:`A` parameter
    `B`             :class:`Quantity`       The :math:`B` parameter
    =============== ======================= ====================================
    """
```
It would be good if you could include relevant portions here (and also provide the references you mentioned in the enhancement request Cantera/enhancements#113).

Reply:
Updated and will submit website revision shortly